deploy/kubeturbo-operator/helm-charts/kubeturbo/templates/serviceaccount.yaml

```diff
@@ -26,6 +31,15 @@ rules:
       - resourcequotas
```

Comment:
As of now kubeturbo does not need `limitranges` in its capabilities.

Reply:
Confirmed with Enlin, we have a backlog item to support `limitrange` during resize. So I think we should keep this.